templates/single.twig

```diff
@@ -21,7 +21,7 @@
 					{% endif %}
 
 					{% if ( page_type and (post.issues_nav_data or post.tags) ) %}
-						<span class="tag-wrap-bullet" aria-hidden="true">&#8226;</span>
+						<span class="main-tag-chevron" aria-hidden="true"></span>
```

Comment:
You can try `<span class="main-tag-chevron" aria-hidden="true" />` and check if it works, since this element has no children.

Reply:
I tried but that doesn't work properly with twig 😕